server/protocols/webadmin/webadmin-data/src/test/java/org/apache/james/webadmin/routes/DLPConfigurationRoutesTest.java

```diff
@@ -264,19 +238,20 @@ void putShouldReturnBadRequestWhenIdIsNotSpecified() {
                 .contentType(JSON_CONTENT_TYPE)
                 .body("statusCode", is(400))
                 .body("type", is("InvalidArgument"))
-                .body("message", is("JSON payload of the request is not valid"));
+                .body("message", is("JSON payload of the request is not valid"))
+                .body("details", is("Instantiation of [simple type, class org.apache.james.webadmin.dto.DLPConfigurationItemDTO] value failed: 'id' is mandatory (through reference chain: org.apache.james.webadmin.dto.DLPConfigurationDTO[\"rules\"])"));
```

Comment:
This detail is not okay, I need some suggestions

Reply:
No it seems fine to me